src/components/containers/EventCard/chips/StartDateChip.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react'
 import styles from "./chipStyles.module.css";
 import ClockIcon from "../../../../assets/images/icons/clock.svg";
+import dayjs, {ConfigType, Dayjs} from "dayjs";
 
 interface StartDateChipProps {
-    date: Date
+    date: ConfigType | Dayjs
 }
 
-export const StartDateChip: React.FC<StartDateChipProps> = ({date}) => (
-    <div className={styles.chip}>
-        <ClockIcon/>
-        <span className={styles.text}>
-            {date.toDateString()}
-        </span>
-    </div>
-)
\ No newline at end of file
+export const StartDateChip: React.FC<StartDateChipProps> = ({date}) => {
+    const startDate = dayjs(date)
+
+    return (
+        <div className={styles.chip}>
+            <ClockIcon/>
+            <span className={styles.text}>
+                {/* display date with year only if year is different for current*/}
+                {startDate.format(`D MMMM ${startDate.year() !== dayjs().year() ? 'YYYY' : ''}, HH:mm`)}
```

Comment:
should be extracted to some helper function

Reply:
i moved it above the jsx return i don't think it's necessary to separate this from component
it won't be used anywhere else anyways